client/src/test/java/io/spine/client/CommandFactoryTest.java

```diff
@@ -122,6 +96,7 @@ void withOwnTenantId() {
     @DisplayName("throw ValidationException when creating command")
     class NotAccept {
 
+        @SuppressWarnings("unused")
         private final RequiredFieldCommand invalidCommand =
```

Comment:
In fact, it _may_ used in the methods below.

Please see those. They both have the same _local_ variable defined. So maybe we could kill those local variables instead. And remove this suppression.